Snowflake.Data.Tests/IntegrationTests/SFPutGetTest.cs

```diff
@@ -496,23 +521,51 @@ private void GetFile(DbConnection conn)
                     reader.GetString((int)SFResultSet.PutGetResponseRowTypeInfo.ResultStatus));
 
                 // Check file contents
-                using (var streamReader = new StreamReader(t_outputFilePath))
+                foreach (var line in ReadOutputFileLinesUnzippingIfCompressedBecauseOfAutoCompression())
```

Comment:
stream is no longer related to the file path given previously, why?

Reply:
renamed to ReadOutputFileLinesUnzippingIfReallyCompressed() to be a little shorter.
It is related to the t_outputFilePath, but giving it as parameter doesn't give us anything. It is quite obvious that ReadOutputFile...() reads output file, and the name of the output file is stored on t_outputFilePath. If the method would be named ReadFile(), then you would be right than a parameter to point the path would be required.

Reply:
It is not clear when a parameter to a method is indirect and comes from a class member. Also we just need to read lines from file. Why does it matter if file is compressed?

Reply:
changed to ReadOutputFileLines()